moja-kuhinjica-frontend/src/components/restaurant/restaurant_card/RestaurantCard.tsx

```diff
@@ -0,0 +1,84 @@
+import Image from 'next/image'
+
+const RestaurantCard = () => {
+    return (
+        <Wrapper
+            height="330px"
+            width="430px"
+            justifyContent="center"
+            alignItems="center"
+        >
+            <ColDiv width="90%">
+                <RestaurantPic src={pic} alt="" />
+                <RowDiv>
+                    <ColDiv justifyContent="center">
+                        <TitleLabel
+                            size={'20px'}
+                            content={'Restoran Top FOOD 021'}
+                        />
+                        <RowDiv>
+                            <RowDiv width="60%" alignItems="center">
+                                <RegularLabel
+                                    size={'13px'}
+                                    content={'RESTORANI'}
+                                />
+                                <LabelSeparator>&bull;</LabelSeparator>
+                                <RegularLabel
+                                    size={'13px'}
+                                    content={'Domaća kuhinja'}
+                                />
+                            </RowDiv>
+                            <RowDiv
+                                justifyContent="right"
+                                width="40%"
+                                alignItems="center"
+                            >
+                                <RegularLabel
+                                    size={'13px'}
+                                    content={'0.5 km'}
+                                />
+                                <LabelSeparator>&bull;</LabelSeparator>
+                                <RegularLabel
+                                    size={'13px'}
+                                    content={'15 min'}
+                                />
+                            </RowDiv>
+                        </RowDiv>
+                        <RowDiv>
+                            <RowDiv alignItems="center">
+                                <RestaurantRating>4.2</RestaurantRating>
+                                <RowDiv alignItems="center">
+                                    <Image src={star} alt=""></Image>
+                                    <Image src={star} alt=""></Image>
+                                    <Image src={star} alt=""></Image>
+                                    <Image src={star} alt=""></Image>
+                                    <Image src={star} alt=""></Image>
+                                </RowDiv>
+                            </RowDiv>
+                            <RowDiv
+                                justifyContent="right"
+                                width="50%"
+                                alignItems="center"
+                            >
+                                <RegularLabel
+                                    size={'13px'}
+                                    content={'Besplatna dostava'}
```

Comment:
When you pass strings as props you can do that without curly brackets

Reply:
Done.

Reply:
Typescript is showing error because `size` in number and you pass string prop